arches/app/models/migrations/2532_boolean_datatype_display_value.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+from django.db import migrations, models
+from django.core import management
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('models', '2402_node_to_node'),
+    ]
+
+    operations = [
+            migrations.RunSQL("""
```

Comment:
We should also add to this migration some SQL to remove these configuration values from `widgets.defaultconfig` (and add them back in reverse)

Reply:
we probably also want to use something other than an empty string for the default labels; it appears that previously we used "Yes" and "No"